renderdoc/driver/gl/gl_debug.cpp

```diff
@@ -1165,7 +1185,7 @@ uint32_t GLReplay::PickVertex(uint32_t eventID, const MeshDisplay &cfg, uint32_t
       (GLsizeiptr)(cfg.position.idxoffs + cfg.position.idxByteWidth * cfg.position.numVerts));
   gl.glBindBufferBase(eGL_SHADER_STORAGE_BUFFER, 3, DebugData.pickResultBuf);
 
-  gl.glDispatchCompute(GLuint(cfg.position.numVerts / 128 + 1), 1, 1);
+  gl.glDispatchCompute(GLuint((cfg.position.numVerts / 3) / 128 + 1), 1, 1); //launch one thread per triangle
```

Comment:
Yes that's right, this wouldn't function with other kinds of topologies. Seems straightforward to fix though. I'll try updating the CS to support the other triangle topologies, plus a 'fallback' path for points/lines. The PickResult logic will have to have separate paths for triangles and points/lines too.